tests/tools/onert_train/src/onert_train.cc

```diff
@@ -138,11 +138,16 @@ int main(const int argc, char **argv)
     NNPR_ENSURE_STATUS(nnfw_train_get_traininfo(session, &tri));
 
     // overwrite training information using the arguments
-    tri.batch_size = args.getBatchSize();
-    tri.learning_rate = args.getLearningRate();
-    tri.loss_info.loss = args.getLossType();
-    tri.loss_info.reduction_type = args.getLossReductionType();
-    tri.opt = args.getOptimizerType();
+    if (tri.batch_size == 0 && !args.getBatchSize().has_value())
```

Comment:
AFAIU, now `tri` has train info including `batch_size`, `learning_rate` and ...

Reply:
Why is `batch_size` being handled differently?

Reply:
@jyoungyun gives me a idea about batch_size default value in the draft : https://github.com/Samsung/ONE/pull/12445/files#r1461482143  

> (suggestion)
> How about setting a default value for batch_size? Since the batch size is not related to the model file, onert_train can provide a default batch size value to the user.

> TensorFlow sets 32 as the default batch size, and PyTorch sets 1 as the default batch size. In ONERT, if the data length is not divisible by the batch size, it does not train the rest of the dataset. Therefore, I suggest setting 1 as the batch size for onert_train. 


Hmm.. but rethinking this part, I'd better talk about this with @jyoungyun once again.

Reply:
Rethinking about this part, I changed my mind a bit. 

To give a default value to the batch_size, Maybe we'd better update IR::TrainingInfo has a default value. 
https://github.com/Samsung/ONE/blob/356901d7a13c270252919834b581096e2cf35663/runtime/onert/core/include/ir/train/TrainingInfo.h#L35 

To give a default value on batch_size, there are places to inject the default value. 
 * make onert_train/Args has a default value
     But if we set a default value on Args class. It is hard to distinguish default value and user given value. 

 * Second, make onert_train.cc set a default value if both model file and cmd opt aren't given 
     But it just changes one tool's policy, not the training core. 

  * Third, update ir::TrainingInfo constructor to set a default value.
     This looks best to me.

Reply:
@jyoungyun I'd be grateful if you left any opinion about this. 
Since it is tricky to discuss online, I'm also welcome to have some talk on offline :smile:

Reply:
> (suggestion)
> How about setting a default value for batch_size? Since the batch size is not related to the model file, onert_train can provide a default batch size value to the user.
> 
> TensorFlow sets 32 as the default batch size, and PyTorch sets 1 as the default batch size. In ONERT, if the data length is not divisible by the batch size, it does not train the rest of the dataset. Therefore, I suggest setting 1 as the batch size for onert_train.

A few weeks ago, I suggested to use default value for the batch_size. This is because it is easier to use the `onert_train` tool for users. However, if the model file is circle file rather than circle+ file, the user must provide many parameters. I think it is better that the batch_size is handled like other paramters. Let's remove the default value of batch_size!